Docs/Book/C++Examples/example11.cpp

```diff
@@ -0,0 +1,52 @@
+//
+// Working with 3D molecules - example11.cpp
+
+#include <iostream>
+#include <map>
+#include <string>
+#include <vector>
+#include <Geometry/point.h>
+#include <GraphMol/GraphMol.h>
+#include <GraphMol/MolOps.h>
+#include <GraphMol/MolAlign/AlignMolecules.h>
+#include <GraphMol/DistGeomHelpers/Embedder.h>
+#include <GraphMol/ForceFieldHelpers/MMFF/MMFF.h>
+#include <GraphMol/ForceFieldHelpers/UFF/UFF.h>
+#include <GraphMol/SmilesParse/SmilesParse.h>
+
+int main( int main , char **argv ) {
+
+  RDKit::ROMOL_SPTR mol( RDKit::SmilesToMol( "C1CCC1OC" ) );
+  RDKit::ROMOL_SPTR mol1( RDKit::MolOps::addHs( *mol ) );
+  // Original distance geometry embedding
+  RDKit::DGeomHelpers::EmbedMolecule( *mol1 , 0 , 1234 );
+  RDKit::UFF::UFFOptimizeMolecule( *mol1 );
+  
+  // new Riniker and Landrum CSD-based method
```

Comment:
I did that, as a separate example lower in the file, and also in the text, with explanation.  Granted, I did
params.randomSeed = true;
which was clearly a mistake.  If you're not seeing that, then something has gone awry with the push.  I will correct the bug and check on the new push.